internal/cmd/controller/reconciler/cluster_controller.go

```diff
@@ -170,8 +169,12 @@ func (r *ClusterReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ct
 	}
 
 	if allReady && cluster.Status.ResourceCounts.Ready != cluster.Status.ResourceCounts.DesiredReady {
-		logrus.Debugf("Cluster %s/%s is not ready because not all gitrepos are ready: %d/%d, enqueue cluster again",
-			cluster.Namespace, cluster.Name, cluster.Status.ResourceCounts.Ready, cluster.Status.ResourceCounts.DesiredReady)
+		logger.V(1).Info("Cluster is not ready, because not all gitrepos are ready",
+			"namespace", cluster.Namespace,
+			"name", cluster.Name,
+			"ready", cluster.Status.ResourceCounts.Ready,
+			"DesiredReady", cluster.Status.ResourceCounts.DesiredReady,
```

Comment:
Just curious: is there a rationale for upper- vs lowercase first chars of log fields here?
Are they deliberately inconsistent between themselves for the sake of keeping `DesiredReady` consistent with the state name it refers to?

Reply:
Right, should be `desiredReady`.

Reply:
```suggestion
			"desiredReady", cluster.Status.ResourceCounts.DesiredReady,
```